### PR TITLE
Implement DQN agent.

### PR DESCRIPTION
**Pull request by Ethan-Wolfe**

### CODE REVIEW

**Comment by haraldger** [comment]
Requires 2 fixes:
- add matplotlib to the yml file
- Fix typo in readme, should be "conda activate mario" instead of "myenv"

File: DQN/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+1. Install [conda](https://www.anaconda.com/products/individual)
+2. Install dependencies with `environment.yml`
+    ```
+    conda env create -f environment.yml
+    ```
+    Check the new environment *mario* is [created successfully](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file).
+
+3. Activate *mario* enviroment
+    ```
+    conda activate myenv
```
Comment (haraldger) [inline DQN/README.md]: change to "conda activate mario"

File: DQN/environment.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: mario
```
Comment (haraldger) [inline DQN/environment.yml:1]: Add matplotlib as dependency

**Review by haraldger — CHANGES_REQUESTED** [review]

File: DQN/README.md (hunk 1 above)
Comment (haraldger) [reply DQN/README.md:11]: When I install matplotlib it says it requires pyglet<=1.5.0,>=1.4.0, although yml file installs pyglet 1.5.5, incompatible

Comment (haraldger) [reply DQN/README.md:11]: Missing dependency skimage

File: DQN/environment.yml (hunk 2 above)
Comment (haraldger) [reply DQN/environment.yml:1]: When I install matplotlib it says it requires pyglet<=1.5.0,>=1.4.0, although yml file installs pyglet 1.5.5, incompatible

Comment (haraldger) [reply DQN/environment.yml:1]: When I install matplotlib it says it requires pyglet<=1.5.0,>=1.4.0, although yml file installs pyglet 1.5.5, incompatible

Comment (haraldger) [reply DQN/environment.yml:1]: Missing dependency skimage, or "scikit-image" in pip

File: DQN/README.md (hunk 1 above)
Comment (Ethan-Wolfe) [reply DQN/README.md]: Done

**Comment by Ethan-Wolfe** [comment]
Done

**Review by haraldger — APPROVED** [review]
This is probably adequate, I'm like 3 long island iced teas down

**Pull request merged**